### PR TITLE
OFFAPPS-453 split out notes permission from organization editing

### PR DESCRIPTION
**Pull request by sandlerr**
:koala:

org notes editing is a separate permission.

/cc @zendesk/quokka

### References
 - Jira link: https://zendesk.atlassian.net/browse/OFFAPPS-453

### Risks
 - None

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -399,9 +397,16 @@
       var role = _.find(roles, function(role) {
         return role.id == this.currentUser().role();
       }, this);
-      this.orgEditable = role.configuration.organization_editing;
+      this.orgEditable.general = role.configuration.organization_editing;
+      this.orgEditable.notes = role.configuration.organization_notes_editing;
       _.each(this.storage.organizationFields, function(field) {
-        field.editable = field.editable !== undefined ? this.orgEditable : undefined;
+        if (field.key === '##builtin_tags') {
+          return;
```
Comment (adammw) [inline app.js]: as in everyone can edit tags? or no-one can?

**Comment by adammw** [comment]
:+1: apart from my lack of understanding around tags.

File: app.js (hunk 1 above)
Comment (sandlerr) [inline app.js]: nobody can edit them in this app. Its the same for user and org tags - I guess we would need UI components in here to do that nicely.

**Comment by svizzari** [comment]
:+1:

**Pull request merged**